mm2src/coins/z_coin/z_coin_errors.rs

```diff
@@ -423,6 +423,8 @@ pub enum ZcoinStorageError {
     NotSupported(String),
     #[cfg(target_arch = "wasm32")]
     ZcashParamsError(String),
+    #[cfg(target_arch = "wasm32")]
+    MalfunctionedParamsData,
```

Comment:
Why was this error variant added?

Reply:
leftover from this comment. https://github.com/KomodoPlatform/komodo-defi-framework/pull/1996#discussion_r1434443082

removed